oxidation/bindings/README.md

```diff
@@ -54,5 +54,6 @@ Build:
 
 ```bash
 # In /bindings/web
-wasm-pack build
```

Comment:
Nope

Reply:
What happened ? I changed it in my local branch

I also noticed a force push, did I mess up something ?

Reply:
I modified